.github/workflows/mirror.yml

```diff
@@ -0,0 +1,18 @@
+name: Mirror
+
+on: push
```

Comment:
Should it be limited to the `master` branch?

Reply:
That's a good thought! I'll limit it to the default branch. 👍